HW04FTP/CLIENT/Network.cs

```diff
@@ -0,0 +1,86 @@
+using System.Net.Sockets;
+
+namespace Client;
+
+/// <summary>
+/// Managing connection with server
+/// </summary>
+public class ClientNetwork
+{
+    private int _port;
+    private string _address;
+    private TcpClient _client;
+
+    private TcpClient GetTcpClient()
+    {
+        TcpClient client;
+        try
+        {
+            client = new TcpClient();
+            return client;
+        }   
+        catch (Exception ex)
+        {
+            Console.WriteLine($"Cannot connect to {_port} on {_address} address");
+            Console.WriteLine($"Exception message: {ex.Message}");
+        }
+        throw new Exception("Client stayed unassigned for some reason");
+    }
+    public ClientNetwork()
+    {
+        _port = 1337;
+        _address = "localhost";
+        _client = GetTcpClient();
+        _client.ConnectAsync(_address, _port);
+    }
+
+    public ClientNetwork(string address)
+    {
+        _port = 1337;
+        _address = address;
+        _client = GetTcpClient();
+
+    }
```

Comment:
```suggestion
    public ClientNetwork(string address)
    {
        _port = 1337;
        _address = address;
        _client = GetTcpClient();
    }
```
лишняя пустая строка